Build failure on macos main

Building locally I am seeing

```
   Compiling desktop-tui v0.1.1 (/Users/jayvdb/rust/desktop-tui)
error[E0599]: no variant or associated item named `NcursesTerminal` found for enum `appcui::backend::Type` in the current scope
  --> src/main.rs:30:39
   |
30 |     let app = App::with_backend(Type::NcursesTerminal)
   |                                       ^^^^^^^^^^^^^^^ variant or associated item not found in `appcui::backend::Type`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `desktop-tui` (bin "desktop-tui") due to 1 previous error
error: failed to compile `desktop-tui v0.1.1 (/Users/jayvdb/rust/desktop-tui)`, intermediate artifacts can be found at `/Users/jayvdb/rust/desktop-tui/target`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -12,7 +12,7 @@ repository = "https://github.com/Julien-cpsn/desktop-tui"
 keywords = ["Desktop", "Environment", "TUI", "AppCUI-rs"]
 
 [dependencies]
-appcui = { version = "0.3.5", features = ["CROSSTERM", "TRUE_COLORS"] }
+appcui = { version = "0.3.6", features = ["CROSSTERM", "TRUE_COLORS"] }
 
 virtual-terminal = "0.1.4"
 

